src/test/java/util/xcmailr/util/SendEmail.java

```
@@ -0,0 +1,78 @@
+    public static void send(EmailAccount emailAccount, String recipent, String subject, String text)
+    {
+        final Properties smtpProps = new Properties();
+        smtpProps.setProperty("mail.smtp.ssl.enable", Boolean.toString(emailAccount.isSsl()));
+        smtpProps.setProperty("mail.smtp.tls.enable", Boolean.toString(emailAccount.isTls()));
+        smtpProps.put("mail.smtp.host", emailAccount.getServer());
+        smtpProps.put("mail.smtp.auth", "true");
+        smtpProps.put("mail.stmp.port", emailAccount.getPort());
+        smtpProps.setProperty("mail.user", emailAccount.getLogin());
+
+        // create session
+        final SmtpAuthenticator smtpAuthenticator = new SmtpAuthenticator(emailAccount.getLogin(), emailAccount.getPassword());
+        final Session session = Session.getInstance(smtpProps, smtpAuthenticator);
+
+        // Create the message part
+        final MimeMessage message = new MimeMessage(session);
+
+        try
+        {
+
+            final Address[] addresses =
+            {
+              new InternetAddress(recipent)
+            };
+            final BodyPart messageBodyPartText = new MimeBodyPart();
+            messageBodyPartText.setText(text);
+            final BodyPart messageBodyPartHtml = new MimeBodyPart();
+            messageBodyPartHtml.setContent(text, "text/html; charset=utf-8");
+            final Multipart multipart = new MimeMultipart();
+            multipart.addBodyPart(messageBodyPartText);
+            multipart.addBodyPart(messageBodyPartHtml);
+
+            message.setRecipients(Message.RecipientType.TO, addresses);
+            message.setSubject(subject);
+            message.setFrom(new InternetAddress(emailAccount.getEmail()));
+            message.setContent(multipart);
+
+            // Send message
+            Transport.send(message);
+            session.getTransport().close();
+        }
+        catch (final MessagingException e)
+        {
+            e.printStackTrace();
```

The method should throw the `MessagingException` instead of simply printing the stack trace so we know that sending an e-mail failed instead of "silently" ignoring it.